### DISCUSSION TITLE
How to setup an Android/iOS Qt6 project?

### DISCUSSION THREAD

**Discussion by jhay06** [zxing-cpp/zxing-cpp#559] [Q&A]
Hi I try to integrate zxing-cpp in my Android QT project i put the clone zxing-cpp folder to my "imports" folder 
and in my CMakeLists 

add_subdirectory(zxing-cpp)

then i use the example ZXingQt6Camreader example

but i always ended up to an error ld: error: unable to find library -lZXingQtCamReader

**Comment by axxel** [comment]
There is no library `ZXingQtCamReader`, so wonder this can not be found. It is also unclear what you mean my "I use the ZXingQt6Camreader example".

**Comment by jhay06** [comment]
i use the code from Example folder that can be found on the zxing-cpp repo . there is ZXingQt6CamReader there.

**Comment by jhay06** [comment]
i will really appreciate if someone send an example repo for complete setup of Qt6 android / ios app with zxing-cpp thanks

**Comment by jhay06** [comment]
btw , using qt6.5.0. when starting the scan , it crashed immediately ,

**Comment by emericg** [comment]
Hi, here is a full demo app for Qt6 (tested with 6.5.3 and 6.6.1) for iOS and Android.

https://github.com/emericg/QmlMobileScanner

It has backends for qzxing, zxingcpp and zint. The qzxing has been modified to work on Qt6, the zxingcpp and zint backends are "new" but inspired from existing code.
